app/http.php

```diff
@@ -0,0 +1,1039 @@
+        $activeRuntimes = $register->get('activeRuntimes');
+
+        /**
+         * Remove residual runtimes
+         */
+        Console::info('Removing orphan runtimes...');
+        try {
+            $connection = $orchestrationPool->pop();
+            $orchestration = $connection->getResource();
+            $orphans = $orchestration->list(['label' => 'openruntimes-executor=' . System::getHostname()]);
+        } finally {
+            isset($connection) && $orchestrationPool->push($connection);
+        }
+
+        if (\count($orphans) === 0) {
+            Console::log("No orphan runtimes found.");
+        }
+
+        $callables = [];
+        foreach ($orphans as $runtime) {
+            $callables[] = function () use ($runtime, $orchestrationPool) {
+                try {
+                    $connection = $orchestrationPool->pop();
+                    $orchestration = $connection->getResource();
+                    $orchestration->remove($runtime->getName(), true);
+                    Console::success("Successfully removed {$runtime->getName()}");
+                } catch (\Throwable $th) {
+                    Console::error('Orphan runtime deletion failed: ' . $th->getMessage());
+                } finally {
+                    isset($connection) && $orchestrationPool->push($connection);
+                }
+            };
+        }
+
+        /** @phpstan-ignore-next-line */
+        Swoole\Coroutine\batch(
+            $callables
+        );
+
+        Console::success("Done.");
+
+        /**
+         * Warmup: make sure images are ready to run fast 🚀
+         */
+        Console::info('Pulling runtime images...');
+        $runtimes = new Runtimes('v2');
+        $allowList = empty(App::getEnv('OPEN_RUNTIMES_EXECUTOR_RUNTIMES')) ? [] : \explode(',', App::getEnv('OPEN_RUNTIMES_EXECUTOR_RUNTIMES'));
+        $runtimes = $runtimes->getAll(true, $allowList);
+        $callables = [];
+        foreach ($runtimes as $runtime) {
+            $callables[] = function () use ($runtime, $orchestrationPool) {
+                try {
+                    $connection = $orchestrationPool->pop();
+                    $orchestration = $connection->getResource();
+                    Console::log('Warming up ' . $runtime['name'] . ' ' . $runtime['version'] . ' environment...');
+                    $response = $orchestration->pull($runtime['image']);
+                    if ($response) {
+                        Console::info("Successfully Warmed up {$runtime['name']} {$runtime['version']}!");
+                    } else {
+                        Console::warning("Failed to Warmup {$runtime['name']} {$runtime['version']}!");
+                    }
+                } finally {
+                    isset($connection) && $orchestrationPool->push($connection);
+                }
+            };
+        }
+
+        /** @phpstan-ignore-next-line */
+        Swoole\Coroutine\batch(
```

Comment:
Why don't we `use function Swoole\Coroutine\batch` and use `batch` directly?  

Then we can get rid of the phpstan ignore line

Reply:
PHPStan still gives `Used function Swoole\Coroutine\batch not found.`.

But one ignore is better than 4. Adding `use`, making this `batch()`.

Reply:
Figured it out, needed to tell PHP Stan about Swoole IDE helper.